charts/artifact-hub/values.schema.json

```diff
@@ -270,6 +270,24 @@
                                                     "default": ""
                                                 }
                                             }
+                                        },
+                                        "resources": {
+                                            "title": "Hub pod resource requirements",
```

Comment:
Can you replace `Hub` for `DB migrator` or something similar? Hub is actually one of the components :) Same for the DB readiness one please.

Reply:
Argh ... sure, sorry.

Reply:
Thanks, no problem 👍

Reply:
I changed the titles to a context specific text